editor/editor_properties.cpp

```diff
@@ -3701,6 +3677,36 @@ void EditorPropertyNodePath::_notification(int p_what) {
 
 void EditorPropertyNodePath::_bind_methods() {
 }
+Node *EditorPropertyNodePath::get_base_node() {
+	Node *base_node = nullptr;
+	if (base_hint != NodePath()) {
+		if (get_tree()->get_root()->has_node(base_hint)) {
```

Comment:
Can be merged into 1 condition.
```suggestion
	if (!base_hint.is_empty() && get_tree()->get_root()->has_node(base_hint)) {
```